commitlint.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  parserPreset: './parser-preset.js',
+  rules: {
+    'type-enum': [2, 'always', [`FRDQ`]],
+    'commit-message-rule': [2, 'always'],
+    'subject-empty': [2, 'never'],
+    'subject-case': [2, 'always', 'sentence-case'],
+  },
+  plugins: [
+    {
+      rules: {
+        'commit-message-rule': ({ type, id }) => {
+          const PREFIX = `FRDQ`;
```

Comment:
Could we move this prefix into some global file? Then it could be used by other users from other projects

Reply:
Do you mean to create PREFIX.js in the root directory? Or separate as a repo?

Reply:
Maybe something like globals.js in the root directory or .env?